elixir/apps/web/lib/web/live/resources/show.ex

```diff
@@ -96,10 +96,7 @@ defmodule Web.Resources.Show do
         <span :if={not is_nil(@resource.deleted_at)} class="text-red-600">(deleted)</span>
       </:title>
       <:action :if={is_nil(@resource.deleted_at)}>
-        <.edit_button
-          :if={Domain.Accounts.multi_site_resources_enabled?(@account)}
```

Comment:
I realized that we weren't allowing editing the resource at all, so this was removed to allow it. The actual form contains logic for multi-site enabled/disabled, so this allows editing the Resource name, site, and traffic filters even when multi-site is disabled.

Reply:
James was complaining about not being able to edit typo'd Resource names, or wanting to rename the Resource refs #2504

Reply:
Somewhat related to this, are we intentionally allowing multi-site resource editing on Prod right now?

Reply:
> are we intentionally allowing multi-site resource editing on Prod right now?

We aren't allowing any resource editing on Prod right now.

This is to fix that to allow resource editing, which includes selecting a different site for it. I think that's OK since the form doesn't allow assigning multiple sites (multi-site is still disabled), only changing it.

Reply:
Here's what I see on Prod right now

<img width="1231" alt="Screenshot 2024-05-06 at 12 34 03 PM" src="https://github.com/firezone/firezone/assets/2646332/fc903ed5-745e-459d-a323-b378be08740b">

Reply:
Resolved in standup